server/routerlicious/packages/routerlicious/src/deli/index.ts

```diff
@@ -72,6 +72,11 @@ export async function deliCreate(
 	core.DefaultServiceConfiguration.deli.enableEphemeralContainerSummaryCleanup =
 		enableEphemeralContainerSummaryCleanup;
 
+	const ephemeralContainerSoftDeleteTimeInMs =
+		(config.get("deli:ephemeralContainerSoftDeleteTimeInMs") as number | undefined) ?? -1; // -1 means not soft deletion but hard deletion directly
```

Comment:
nit: maybe change var to ephemeralContainerPurgeAfterSoftDeleteInMs

Reply:
for its value, either 3 or 5 days sound good to me

Reply:
I will make mistake for that for sure, will just use this settings for now